src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/TestProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests
+{
+    public class TestProfile
+    {
+        public string Oid { get; set; }
+        public string FullName { get; set; }
+        public bool IsAppToken { get; set; } = false;
+        public List<string> Roles { get; set; } = new List<string>();
```

Comment:
Is Roles here UserRoles? (Could be confused witth functional roles since we have so many references to it in IPO)

Reply:
It's taken from Fusion. I think this simulate how a Bearer token is serialized and Roles represent a RoleClaim. Won't fix since I'm not sure if a break something